src/test/java/io/zold/api/WalletTest.java

```diff
@@ -61,10 +56,15 @@ public void readsWalletId() throws IOException {
         MatcherAssert.assertThat(wallet.id(), Matchers.is(id));
     }
 
-    @Ignore
     @Test
-    public void throwIoExceptionIfReadingIdFails() throws IOException {
-        this.error.expect(IOException.class);
+    public void throwRuntimeExceptionIfReadingIdFails() throws IOException {
+        this.error.expect(RuntimeException.class);
+        new Wallet.File(this.wallet("")).id();
+    }
+
+    @Test
+    public void throwRuntimeExceptionIfIdIsInvalid() throws IOException {
+        this.error.expect(RuntimeException.class);
```

Comment:
@paulodamaso same here. Let's expect a narrower exception than `RuntimeException`. Furthermore, let's on check the exception message with `this.error.expectMessage` as well.

Reply:
@carlosmiranda I've used `FileNotFoundException`, but I can't check its message. I think it's OS and locale related, for me it appears `(Acesso Negado)` instead of  `(Acces Denied)`. I think that it's enough just narrowing the type.